### PR TITLE
Undo redo [Logger]

### PR DESCRIPTION
**Pull request by iamrakesh28**
Fixes convict-git#89

### CODE REVIEW

File: src/processing/UndoRedo.java [hunk 1]
```diff
@@ -214,8 +272,9 @@ private static void deleteFromStack(ArrayList <BoardObject> stack, ObjectId obje
 		   ListIterator <BoardObject> iterStack = stack.listIterator();
 		   while (iterStack.hasNext()) {
 			   BoardObject obj = iterStack.next();
-			   if (obj.getObjectId().equals(objectId))
+			   if (obj.getObjectId().equals(objectId)) {
 				   iterStack.remove();
+			   }
 		   }
 		   return;
```
Comment (sakshi1603) [inline src/processing/UndoRedo.java]: Fix indentation between this and next function.

Comment (iamrakesh28) [reply src/processing/UndoRedo.java]: Fixed

File: src/processing/UndoRedo.java [hunk 2]
```diff
@@ -149,53 +187,55 @@ private static BoardObject rotateOperation(BoardObject obj, Angle angle) {
...
+	private static BoardObject undoRedoUtil(
 			ArrayList <BoardObject> curStack,
 			ArrayList <BoardObject> otherStack,
 			Operation operation
 	) {
 		
 		// No undo or redo operation possible
     	if (curStack.size() <= 0)
-    		return;
+    		return null;
```
Comment (sakshi1603) [inline src/processing/UndoRedo.java:200]: Include logs for this case and also add this in doc string.

File: src/processing/UndoRedo.java (hunk 1 above)
Comment (sakshi1603) [reply src/processing/UndoRedo.java]: I cant see it updated. Can you check once?

Comment (iamrakesh28) [reply src/processing/UndoRedo.java]: I didn't notice this and I fixed something else. Now, I have fixed it. Check it once.

**Review by RathoreDevansh08 — APPROVED** [review]

**Pull request merged**